scripts/generate_psa_key_attributes/generate_psa_key_attributes.rst

```diff
@@ -0,0 +1,210 @@
+.. _generate_psa_key_attributes_script:
+
+PSA Key Attributes generator script
+###################################
```

Comment:
Please add this also to the code owners file and assign to ncs-aegir -doc. Also, this file has not been added here: https://ncsdoc.z6.web.core.windows.net/PR-23113/nrf/scripts.html

Reply:
> It was added, here: [2df9b1d#diff-52e21384533bdbbb2558cbf70c8daef03760d8df203a1785416ca66c87a937f7](https://github.com/nrfconnect/sdk-nrf/commit/2df9b1dca95dd5705bd4ad39b52e761d490497bc#diff-52e21384533bdbbb2558cbf70c8daef03760d8df203a1785416ca66c87a937f7)
> 
> And this commit is in this PR, so all good.

Ok..probably the preview is not updated then.